eclair-core/src/test/java/fr/acinq/eclair/MilliSatoshiTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 ACINQ SAS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.acinq.eclair;
+
+import fr.acinq.bitcoin.Satoshi;
+
+/**
+ * This class is a compile-time check that we are able to compile Java code that uses MilliSatoshi utilities.
+ */
+public final class MilliSatoshiTest {
```

Comment:
I think that if we want real Java tests we'll need to add JUnit support (akka is using JUnit for their Java tests)

Reply:
Just making sure that it compiles seems good enough for now.